client/js/README.md

```diff
@@ -21,9 +21,9 @@ The client function connects to the API of a hosted Gradio space and returns an
 The simplest example looks like this:
```

Comment:
"The JavaScript Gradio Client exposes two named imports, client and duplicate" needs to be updated